StanleyParableXiv/Services/AudioPlayer.cs

```diff
@@ -447,34 +463,66 @@ public class AudioPlayer : IDisposable
     /// Service used for playing audio files from the Resources folder.
     /// Audio mixer implementation referenced from https://github.com/Roselyyn/EldenRingDalamud
     /// </summary>
-    public AudioPlayer()
+     private AudioPlayer()
     {
-        _outputDevice = new WaveOutEvent();
-        _mixer = new MixingSampleProvider(WaveFormat.CreateIeeeFloatWaveFormat(44100, 2))
-        {
-            ReadFully = true
-        };
-        _sampleProvider = new VolumeSampleProvider(_mixer);
-
-        _mixer.MixerInputEnded += OnMixerInputEnded;
-        
-        _outputDevice.Init(_sampleProvider);
-        _outputDevice.Play();
-        
-        UpdateVolume();
+        InitializeAudioDevice();
     }
 
-    public void Dispose()
+    private void InitializeAudioDevice()
     {
-        _outputDevice.Dispose();
-        GC.SuppressFinalize(this);
-    }
+        try
+        {
+            // Try WaveOut first
+            _outputDevice = new WaveOutEvent();
+            _mixer = new MixingSampleProvider(WaveFormat.CreateIeeeFloatWaveFormat(44100, 2))
+            {
+                ReadFully = true
+            };
+            _sampleProvider = new VolumeSampleProvider(_mixer);
 
+            _mixer.MixerInputEnded += OnMixerInputEnded;
+            _outputDevice.Init(_sampleProvider);
+            _outputDevice.Play();
+            _isInitialized = true;
+
+            UpdateVolume();
+        }
+        catch (Exception ex)
+        {
+            DalamudService.Log.Error(ex, "Failed to initialize primary audio device, attempting fallback");
+            
+            try
+            {
+                // Fallback to DirectSound
+                _outputDevice?.Dispose();
+                _outputDevice = new DirectSoundOut();
+                _mixer = new MixingSampleProvider(WaveFormat.CreateIeeeFloatWaveFormat(44100, 2))
+                {
+                    ReadFully = true
+                };
+                _sampleProvider = new VolumeSampleProvider(_mixer);
+
+                _mixer.MixerInputEnded += OnMixerInputEnded;
+                _outputDevice.Init(_sampleProvider);
+                _outputDevice.Play();
+                _isInitialized = true;
+
+                UpdateVolume();
```

Comment:
Code from setting `_mixer` to `UpdateVolume()` is repetitive. Where does the initialization fail? The code could be refactored to it's own method with a type `T` parameter that can accept either `WaveOutEvent` or `DirectSoundOut`. This would also be helpful if this was made into a user configurable option.